otter/integration/tests/test_convergence.py

```diff
@@ -129,6 +178,16 @@ def _choose_half_the_servers(self, (code, response)):
         self.n_killed = self.n_servers / 2
         return ids[:self.n_killed]
 
+    def _choose_random_servers(self, (code, response), n):
+        """Selects ``n`` randomly selected servers from those returned by the
+        ``get_scaling_group_state`` function.
+        """
+
+        if code != 200:
+            raise Exception("Got 404; dude, where's my scaling group?")
```

Comment:
I like the message :-), but the 404 part is inaccurate; the real code should be interpolated here.